jvm-slides/src/main/groovy/org/asciidoctor/gradle/jvm/slides/RevealJSOptions.groovy

```diff
@@ -109,10 +109,18 @@ class RevealJSOptions {
         this.progressBar = Optional.ofNullable(b)
     }
 
-    /** Display the slide number of the current slide.
-     *
+    /**
+     * Display the slider number of the current slide. <br/>
+     * true will display the slide number with default formatting. <br/>
```

Comment:
I'm not sure `true will display` is describing the correct thing.

Reply:
The behavior is a bit weird but `"true"` as a string will display the slide number with default formatting.
I should probably add double quotes and/or point out that `true` is a string?

Reply:
That is a good idea. It should also be one of the enum values.